docs/index.html

```diff
@@ -737,25 +719,25 @@ <h4 class="headline-3">Module: Colors</h4>
 	</section>
 
 	<section class="grid">
-		<div class="grid-row">
-			<div class="grid-cell x3 hint-text">
-				<h4 class="headline-3">Module: Grid - Introduction</h4>
-				<p>
-					The grid is the basic module to provide boxes for all of your modules. It is a global module, which means it is not scoped and can (also should) be used everywhere on the site.
-				</p>
-				<p>
-					The <code>grid-wrapper</code> class provides the white box to hold the module as well as the padding and margin.
-				</p>
+			<div class="grid-row">
+				<div class="grid-cell x3 hint-text">
+					<h4 class="headline-3">Module: Grid - Introduction</h4>
+					<p>
+						The grid is the basic module to provide boxes for all of your modules. It is a global module, which means it is not scoped and can (also should) be used everywhere on the site.
+					</p>
+					<p>
+						The <code>grid</code> class provides a margin around the box and a hidden overflow.
```

Comment:
the grid should be separated and above to make sure people see the difference, because theoretically you could build a module without grid-row. But not a grid-row / grid-col without grid.